assets/app/scripts/services/applicationGenerator.js

```diff
@@ -67,8 +67,8 @@ angular.module("openshiftConsole")
       }
 
       //augment labels
-      input.labels.name = input.name;
-      input.labels.generatedby = "OpenShiftWebConsole";
+      input.labels.app = input.name;
+      input.annotations["openshift.io/generatedby"] = "OpenShiftWebConsole";
```

Comment:
@smarterclayton in all your openshift renaming changes, did you change this annotation name in the CLI?